AiPlatform/.repo-metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "distribution_name": "google/cloud-ai-platform",
+    "release_level": "beta",
```

Comment:
Should this `v1` API start out as `beta`? Just checking (we do the same in Go then promote to GA/stable after some bake time).

Reply:
Same for us, too! Thanks for checking.